com.vrcfury.vrcfury/SPS/sps_search.cginc

```diff
@@ -29,6 +29,13 @@ void sps_parse_light(float range, out int type, int myChannel) {
 		if (legacyRange == 5) type = 3;
 	// }
 
+	// Handle legacy DPS channel 1
+	if (myChannel == -1)	{
```

Comment:
I just noticed, this makes it work with BOTH channel 0 and channel 1, which is probably not right

Reply:
oops, should be fixed now